public/js/responsivemap.js

```diff
@@ -0,0 +1,56 @@
+/*global
+	google
+*/
+
+$(document).ready(function () {
+
+	//Google Maps JS
+	//Set Map
+	function initialize() {
+		var uryLocation = new google.maps.LatLng(53.948193, -1.054030);
```

Comment:
make into a config var?

Reply:
should this be loaded from config.toml or somewhere else? (if so, where?)

Reply:
Oh lol it is, whoops